sqlglot/dialects/postgres.py

```diff
@@ -647,3 +647,6 @@ def cast_sql(self, expression: exp.Cast, safe_prefix: t.Optional[str] = None) ->
                 return self.sql(this)
 
             return super().cast_sql(expression, safe_prefix=safe_prefix)
+
+        def arrayconcat_sql(self, expression: exp.ArrayConcat, name: str = "ARRAY_CAT") -> str:
+            return super().arrayconcat_sql(expression, name=name)
```

Comment:
It's not good practice to change the method signature -- let's instead pass in `"ARRAY_CAT"` in the `super` call.

Reply:
We could also keep the transform entry and do `self.arrayconcat_sql(expression, name=...` instead of overriding the method to use `super`. I think I'd go with this instead.